src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 container">
```

Comment:
```suggestion
    <div class="stars--0 stars">
```

It should be `starts` block, the same in other places